aiohttp/web_response.py

```diff
@@ -160,12 +163,7 @@ def set_status(
             not self.prepared
         ), "Cannot change the response status code after the headers have been sent"
         self._status = int(status)
-        if reason is None:
-            try:
-                reason = HTTPStatus(self._status).phrase
-            except ValueError:
-                reason = ""
-        self._reason = reason
+        self._reason = reason or REASON_PHRASES.get(self._status, "")
```

Comment:
Does the user want to be able to set an empty reason value explicitly? This change would break it.

Reply:
Fixed in 7ec583cbf2fd6f177bbcac9f8e7145fb6168d7c4

Reply:
coverage in a66573560b0f9d0bad9b3574fdb2ee6893f0b22d